### PR TITLE
add sermon detail tags

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -56,3 +63,10 @@ interface StaticPaths {
 	paths: string[];
 	fallback: boolean;
 }
+
+interface RecordingTag {
```
Comment (mattleff) [inline types.d.ts:67]: It would be nice to generate these from the queries.

**Review by mattleff — APPROVED** [review]
LGTM! :shipit:

File: types.d.ts (hunk 1 above)
Comment (narthur) [reply types.d.ts:67]: @mattleff Do you think now is a good time to do that? How would I go about it?

Comment (mattleff) [reply types.d.ts:67]: I think the sooner the better. You could use GraphQL Code Generator with a couple of their plugins: https://graphql-code-generator.com/docs/plugins/typescript-react-query/

**Pull request merged**